src/Cli.Tests/UtilsTests.cs

```diff
@@ -217,6 +247,11 @@ public static void Cleanup()
             {
                 File.Delete("my-config.json");
             }
+
+            if (File.Exists($"{CONFIGFILE_NAME}.merged{CONFIG_EXTENSION}"))
```

Comment:
This also needs to check for environment specific merged file name right? 

in RuntimeConfigPath.cs you have the function GetMergedFileNameForEnvironment return : `$"{fileName}{environmentValue}.merged{CONFIG_EXTENSION}";`